tests/unittests/BackendCorrectnessTest.cpp

```diff
@@ -597,6 +597,18 @@ TEST_P(CPUOnly, groupConvTest) {
   EXPECT_TRUE(out1.isEqual(out2));
 }
 
+/// This test targets the DKKC8 optimization.
+TEST_P(CPUOnly, nonSquarePaddingConvTest) {
+  std::array<size_t, 4> S{{1, 2, 1, 128}};
+  llvm::ArrayRef<size_t> shape(S);
```

Comment:
nit: usually, you can just pass S, and it will be casted to llvm::ArrayRef. No need to explicitly create llvm::ArrayRef